project-base/storefront/pages/categories/[categorySlug].tsx

```diff
@@ -64,12 +62,9 @@ const CategoryDetailPage: NextPage = () => {
                 description={categoryData?.seoMetaDescription}
                 breadcrumbs={categoryData?.breadcrumb}
                 breadcrumbsType="category"
+                isFetchingData={isSkeletonVisible}
             >
-                {isSkeletonVisible ? (
-                    <CategoryDetailPageSkeleton />
-                ) : (
-                    !!categoryData && <CategoryDetailContent category={categoryData} />
-                )}
+                {!!categoryData && <CategoryDetailContent category={categoryData} />}
             </CommonLayout>
```

Comment:
You can easily solve your problem with loading by implementing the changes seen below. However, I think the same problem might be also in other places (where we show skeleton based on fetching but then have another condition on the actual visibility, such as here). In my opinion, those places were not discovered yet only because there is no client-side link to it in the app. I think it should be handled in this PR to get rid of this double problem.


<img width="1159" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/dd5a1fa8-3da7-4577-9407-8b1e260261fe">

Reply:
Nope. Unfortunately flashing is still happening...

Reply:
Yeah, in the end there was one more issue. But should be solved now.